src/main/java/teammates/ui/pagedata/InstructorCourseJoinConfirmationPageData.java

```diff
@@ -26,7 +26,7 @@ public String getInstitute() {
 
     public String getConfirmationLink() {
```

Comment:
The class `InstructorCourseJoinConfirmationPageData ` can be deleted since it is not used?